spec/customer_service_spec.rb

```diff
@@ -0,0 +1,26 @@
+describe CustomerService do
+  let(:twilio)   { class_double(Twilio, :twilio) }
+  let(:client)   { double(:client, messages: messages) }
+  let(:messages) { double(:messages, create: 'text has been sent') }
+  subject        { described_class.new(twilio) }
```

Comment:
Nice use of dependency injection to decouple your classes from their dependencies and allow you to mock Twilio here